solana/solana-ibc/programs/solana-ibc/src/storage.rs

```diff
@@ -308,8 +308,17 @@ pub struct PrivateStorage {
     pub fee_collector: Pubkey,
 
     pub new_fee_collector_proposal: Option<Pubkey>,
+
+    pub assets: map::Map<CryptoHash, Asset>,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Debug)]
```

Comment:
For other types we used borsh instead of the alias so let’s do that I guess?

```suggestion
#[derive(Clone, Debug, borsh::BorshSerialize, borsh::BorshDeserialize)]
```

Reply:
sounds good.